src/main/java/com/eternalcode/core/EternalCore.java

```diff
@@ -98,30 +101,40 @@
 
     private static final String VERSION = Bukkit.getServer().getClass().getName().split("\\.")[3];
 
-    @Getter private ConfigurationManager configurationManager;
-    @Getter private ScoreboardManager scoreboardManager;
-    @Getter private TeleportManager teleportManager;
-    @Getter private LiteCommands liteCommands;
-    @Getter private ChatManager chatManager;
-    @Getter private UserManager userManager;
-    @Getter private Scheduler scheduler;
+    @Getter
+    private ConfigurationManager configurationManager;
+    @Getter
+    private ScoreboardManager scoreboardManager;
+    @Getter
+    private TeleportManager teleportManager;
+    @Getter
+    private LiteCommands liteCommands;
+    @Getter
+    private ChatManager chatManager;
+    @Getter
+    private UserManager userManager;
+    @Getter
+    private Scheduler scheduler;
     private boolean isPaper = false;
+    private BukkitAudiences adventure;
+
 
     @Override
     public void onEnable() {
-        Stopwatch started = Stopwatch.createStarted();
-        Server server = this.getServer();
+        final Stopwatch started = Stopwatch.createStarted();
```

Comment:
Usuń finala, w codestyle unikamy takich rozwiązań, w runtime i tak będzie on finalem!

Reply:
Jak wyżej, przy zapisie mi to automatycznie dodaje 🙄